src/engine/SCons/Tool/applelink.py

```diff
@@ -1,6 +1,6 @@
 """SCons.Tool.applelink
 
-Tool-specific initialization for the Apple gnu-like linker.
+Tool-specific initialization for the Apple's gnu-like linker.
```

Comment:
either _the Apple_ or _Apple's_, but not _the Apple's_

Reply:
Fixed.